lib/galaxy/config.py

```diff
@@ -463,6 +463,12 @@ def __init__( self, **kwargs ):
         self.dynamic_proxy_external_proxy = string_as_bool( kwargs.get( "dynamic_proxy_external_proxy", "False" ) )
         self.dynamic_proxy_prefix = kwargs.get( "dynamic_proxy_prefix", "gie_proxy" )
 
+        self.dynamic_proxy = kwargs.get( "dynamic_proxy", "nodejs" )
+        self.dynamic_proxy_golang_noaccess = kwargs.get( "dynamic_proxy_golang_noaccess", 60 )
```

Comment:
Can we be more future facing here - the fact it is implemented in golang won't be interesting or distinguishing once this is the only proxy - this should probably just be ``dynamic_proxy_noaccess`` as a result?